sdk/identity/identity/test/public/node/azurePipelinesCredential.spec.ts

```diff
@@ -4,22 +4,27 @@
 import { AzurePipelinesCredential } from "../../../src";
 import { isLiveMode } from "@azure-tools/test-recorder";
 import { assert } from "@azure-tools/test-utils";
+import { setLogLevel } from "@azure/logger";
+setLogLevel("verbose");
```

Comment:
nit: remove modifying log level in tests (at least without hooking/restoring it after)

Reply:
done